### ISSUE TITLE
Under-covered topics

### ISSUE THREAD

**Issue by briatte** [briatte/dsr#20]
For an advanced version of the course, some notes on the stuff that were excluded due to time constraints:

1. Data manipulation (would deserve an extra session)
  - Factors, more in depth
  - More merging and pivoting
  - Row-wise operations
  - [New `dplyr` 1.1.0 functions](https://pinboard.in/u:phnk/b:c2a846defb62)
2. Surveys
  - Labelled data (treated in passing; related to the point above re: factors)
  - Survey weights
3. Databases: SQL etc. (with passing mention of big data)
4. APIs and Web scraping
5. Models -- Logit beyond logit: ordered logit, multinomial logit
6. Models -- panel data, FE/RE, standard error corrections
7. Models -- mixed models
8. CVS -- Git/GitHub
9. Dynamic documents -- R Markdown, Quarto
10. Networks (data, viz, models)
11. Text (this time covering e.g. topic models)
12. JavaScript visualization libraries

So, basically, this advanced course could be, over 12 sessions:

- 2 'refresher' sessions to get started again
  - introduce Git/GitHub
  - introduce R Markdown
- 4 more sessions on data
- 1 'refresher' session on (linear) models
- 3 more sessions on models (with a refresher on logit in the first one)
- 1 'extra' session on text
- 1 'extra' session on networks

(Still no space for JS libraries, fair enough.)

This neglects PCA-style stuff and ML, which should probably be its own project, centered on `tidymodels`, covering random forests, gradient-boosted trees, etc.

**Comment by briatte** [comment]
Moved to https://github.com/briatte/dsr/wiki/Advanced-topics
